syscall/hostcalls.c

```diff
@@ -1282,6 +1282,37 @@ oe_result_t _oe_syscall_nanosleep_ocall(
 }
 OE_WEAK_ALIAS(_oe_syscall_nanosleep_ocall, oe_syscall_nanosleep_ocall);
 
+/**
+ * Declare the prototypes of the following functions to avoid the
+ * missing-prototypes warning.
+ */
+oe_result_t _oe_syscall_clock_nanosleep_ocall(
+    int* _retval,
+    int clockid,
+    int flag,
+    struct oe_timespec* req,
+    struct oe_timespec* rem);
+
+/**
+ * Implement the functions and make them as the weak aliases of
+ * the public ocall wrappers.
+ */
+oe_result_t _oe_syscall_clock_nanosleep_ocall(
```

Comment:
add a corresponding test in the edl_opt_out test. See https://github.com/openenclave/openenclave/blob/master/tests/edl_opt_out/enc/enc.c.

Reply:
Added.